meinberlin/apps/kiezradar/serializers.py

```diff
@@ -112,6 +114,8 @@ def to_representation(self, instance):
             {"id": topic.id, "code": topic.code, "name": topics_enum(topic.code).label}
             for topic in instance.topics.all()
         ]
-
         representation["query_text"] = instance.query.text if instance.query else ""
+
+        if not instance.name:
+            representation["name"] = _("Searchprofile %d") % instance.number
```

Comment:
Let's save the object with a name, from the customized save method, and remove this repr here.

Reply:
how do we handle translations then?

Reply:
I didn't think of translations, you are right. Leave it in the serializer then, and we see later.